src/cisco_gnmi/client.py

```diff
@@ -334,3 +340,203 @@ def parse_xpath_to_gnmi_path(self, xpath, origin=None):
             raise Exception("Unfinished elements in XPath parsing!")
         path.elem.extend(path_elems)
         return path
+
+    def combine_configs(self, payload, last_xpath, xpath, config):
```

Comment:
From peer review: Makes sense but immediate use case unclear.
Please provide sample dicts/usage.

Reply:
Refactored this function and it is more readable, added example usage and updated the comment.